content/en/docs/allow-port-80.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Best Practice - Keep Port 80 Open
+permalink: /docs/allow-port-80
+top_graphic: 1
+date: 2019-01-24
+lastmod: 2019-01-24
+---
+
+{{< lastmod >}}
+
+We occasionally get reports from people who have trouble using the
+HTTP-01 challenge type because they've firewalled off port 80 to their
+web server. Our recommendation is that all servers meant for general web
+use should offer both HTTP on port 80 and HTTPS on port 443. They should
+also send redirects for all port 80 requests, and possibly an HSTS header.
+
```

Comment:
How about this?

> It's true that users should be encouraged to interact with your site via HTTPS rather than HTTP. However, closing port 80 doesn't usually provide a security benefit in this regard, compared to other options like redirects and HSTS, but can provide a more confusing experience for site visitors.

Reply:
Are you proposing this in addition to the text above or in place? What is the problem this additional text is trying to solve? I think in general a straightforward "this is our recommendation" is clearer and simpler than trying to imagine every possible objection and counter it. If we do see a lot of people responding to this doc with specific criticisms we can work those in when we find them. But making things longer makes people less likely to read them.

Reply:
I was proposing it as an addition.